index.js

```diff
@@ -11,6 +11,14 @@ var onExit = require('signal-exit')
 var stripBom = require('strip-bom')
 var SourceMapCache = require('./lib/source-map-cache')
 var resolveFrom = require('resolve-from')
+var crypto = require('crypto')
+
+function md5 (str) {
+  return crypto
+    .createHash('md5')
+    .update(str, 'utf8')
+    .digest('hex')
```

Comment:
Personally I'd be happy with this on a single line.

Is md5 still acceptable for these kind of hashes?